src/index.html

```diff
@@ -22,6 +22,382 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="Images/logo.png"
+          alt="MoYo Logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a
+              class="nav-link is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav-item">
+            <a
+              class="nav-link"
+              href="#"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav-item">
+            <a
+              class="nav-link"
+              href="#"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav-item">
+            <a
+              class="nav-link"
+              href="#"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav-item">
+            <a
+              class="nav-link"
+              href="#"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav-item">
+            <a
+              class="nav-link"
+              href="#"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav-item">
+            <a
+              class="nav-link"
+              href="#"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="nav-item">
+            <a
+              class="nav-link"
+              href="#"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <div class="cards__container">
+        <div
+          class="product__card"
+          data-qa="card"
```

Comment:
Since 'product__card' is an element of 'cards__container', it's naming suggests that 'cards' is the block. Therefore, the container should be renamed to 'cards-container' to follow the BEM convention.